packages/shared/src/components/post/collection/CollectionPostContent.tsx

```diff
@@ -0,0 +1,160 @@
+import {
+  PostContainer,
+  PostContentProps,
+  PostNavigationProps,
+} from '../common';
+import Pill from '../../Pill';
+import { CollectionsIntro } from '../widgets';
+
+const CollectionPostContent = ({
+  post,
+  className = {},
+  shouldOnboardAuthor,
+  enableShowShareNewComment,
+  origin,
+  position,
+  inlineActions,
+  onPreviousPost,
+  onNextPost,
+  onClose,
+  postPosition,
+  isFallback,
+  customNavigation,
+  onRemovePost,
+  backToSquad,
+}: PostContentProps): ReactElement => {
+  const { subject } = useToastNotification();
+  const engagementActions = usePostContent({
+    origin,
+    post,
+  });
+  const { updatedAt, contentHtml, image } = post;
+  const { onSharePost: onShare, onReadArticle } = engagementActions;
+
+  const hasNavigation = !!onPreviousPost || !!onNextPost;
+  const containerClass = classNames(
+    'tablet:pb-0 tablet:flex-row',
+    className?.container,
+  );
+
+  const navigationProps: PostNavigationProps = {
+    postPosition,
+    onPreviousPost,
+    onNextPost,
+    post,
+    onReadArticle,
+    onClose,
+    onShare,
+    inlineActions,
+    onRemovePost,
+  };
+
+  return (
+    <PostContentContainer
+      hasNavigation={hasNavigation}
+      className={containerClass}
+      aria-live={subject === ToastSubject.PostContent ? 'polite' : 'off'}
+    >
+      {position === 'fixed' && (
+        <FixedPostNavigation
+          {...navigationProps}
+          className={className?.fixedNavigation}
+        />
+      )}
+
+      <PostContainer
+        className={classNames('relative gap-8', className?.content)}
+        data-testid="postContainer"
+      >
+        {!hasNavigation && (
+          <CollectionPostHeaderActions
+            onShare={onShare}
+            post={post}
+            onClose={onClose}
+            className="flex tablet:hidden"
+            contextMenuId="post-widgets-context"
+          />
+        )}
+        <CollectionsIntro className="tablet:hidden" />
+
+        <BasePostContent
+          className={{
+            ...className,
+            onboarding: classNames(
+              className?.onboarding,
+              backToSquad && 'mb-6',
+            ),
+            navigation: {
+              actions: className?.navigation?.actions,
+              container: classNames('pt-6', className?.navigation?.container),
+            },
+          }}
+          isFallback={isFallback}
+          customNavigation={customNavigation}
+          enableShowShareNewComment={enableShowShareNewComment}
+          shouldOnboardAuthor={shouldOnboardAuthor}
+          navigationProps={navigationProps}
+          engagementProps={engagementActions}
+          origin={origin}
+          post={post}
+        >
+          <Pill
+            label="Collection"
+            className="bg-theme-overlay-float-cabbage text-theme-color-cabbage"
+          />
+
+          <h1
+            className="font-bold break-words typo-large-title"
```

Comment:
```suggestion
            className="font-bold break-words typo-headline"
```
not sure where that came from? Want to understand why we having some discrepancies between design/what we build lately